app.js

```diff
@@ -0,0 +1,37 @@
+let form = document.querySelector('form');
+
+form.addEventListener('submit', async function (event) {
+    event.preventDefault();
+    console.log(form.username.value);
+    let repoNames = await getGitHubRepo(form.username.value);
+    displayRepos(repoNames);
+})
+
+async function getGitHubRepo(username) {
+    try {
+        let resp = await fetch(`https://api.github.com/users/${username}/repos`);
+        let jsonResp = await resp.json();
+        if (resp.ok) {
+
+            let publicRepoNames = jsonResp
+                .filter(repo => repo.private === false)
+                .map(repo => repo.name);
+            return publicRepoNames;
+
+        } else {
+            console.log(jsonResp.message);
+        }
+    } catch (err) {
+        console.log('Something went wrong');
+    }
+}
+
+function displayRepos(repoNames) {
+    repoNames.forEach(repoName => {
+        let ul = document.createElement('ul');
+        let li = document.createElement('li');
+        li.innerHTML = repoName;
+        ul.appendChild(li);
+        document.body.appendChild(ul);
```

Comment:
This creates multiple `ul`s with one `li` each. It should instead create one `ul` with multiple `li`'s.